.gitignore

```diff
@@ -44,5 +44,6 @@ profile
 *.suo
 
 .vs
-out
```

Comment:
Leave it as it was, because it's the default output directory used by Visual Studio for CMake